dist/css/bundle.css

```diff
@@ -0,0 +1,147 @@
+@font-face {
```

Comment:
I'm not sure where this css bundle came from actually, is this old? It didn't work correctly with our charts, went with britecharts.css instead. I'm going to remove it if thats cool.......